requirements/skinny-requirements.txt

```
@@ -7,7 +7,7 @@ cloudpickle<4
 entrypoints<1
 gitpython<4,>=3.1.9
 pyyaml<7,>=5.1
-protobuf<5,>=3.12.0
+protobuf<6,>=3.12.0
```

```suggestion
protobuf<5,>=3.12.0
```

see https://github.com/mlflow/mlflow/pull/11816, i think we still haven't fixed this yet right? cc @WeichenXu123